### ISSUE TITLE
Screen will move up when typing on iOS

### ISSUE THREAD

**Issue by RexS98** [faeibson/lovelace-multiline-text-input-card#5]
When typing on iPhone 11 with iOS 14.1 and updated Homeassistant app, display will move up with typing together.

**Comment by faeibson** [comment]
unfortunately I don't have access to an iPhone. you mean, the "has been saved" message will move the display?

**Comment by jerry-knight** [comment]
No, it scrolls the entire screen up until the text input is not visible.

I have a screen recording here: 

https://youtu.be/9CbrvWTc5Vs

**Comment by faeibson** [comment]
Jerry, it's been a while 😃 During the recent refactorings, restyling, a lot has changed, as has the whole HA itself over the years. Is this still happenning? Sorry I still do not own an iPhone to check myself. Via browser developer options I do not see any erratic behaviour.
